webv2/table/update_column_type.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package table
+
+import (
+	"net/http"
+
+	"github.com/cloudspannerecosystem/harbourbridge/internal"
+	utilities "github.com/cloudspannerecosystem/harbourbridge/webv2/utilities"
+)
+
+//UpdateColumnType updates type of given column to newType.
+func UpdateColumnType(newType, table, colName string, conv *internal.Conv, w http.ResponseWriter) {
+	sp := conv.SpSchema[table]
+
+	//update column type for current table
```

Comment:
Please add a space after // and end the comment with fullstop. Same for all other comments.

Reply:
Updated